docker-compose.yml

```diff
@@ -18,6 +18,22 @@ services:
       - "18250:18250"
     volumes:
       - ./:/blockstore/app/:cached
+    depends_on:
+      - "datastore"
+
+  datastore:
+    container_name: datastore
```

Comment:
For local development, it's probably better to just tell people to run the edX devstack MySQL, which we can use for edx-platform development and blockstore development: https://github.com/edx/devstack/blob/88d22083877005548b31977da99a8092ba1d43b3/docker-compose.yml#L64-L74

Reply:
Good to know, @bradenmacdonald ! Thanks